base/process.jl

```diff
@@ -711,7 +711,8 @@ read(cmd::AbstractCmd, ::Type{String}) = String(read(cmd))
 """
     run(command, args...; wait::Bool = true)
 
-Run a command object, constructed with backticks. Throws an error if anything goes wrong,
+Run a command object, constructed with backticks (see [Running External Programs](@ref)).
```

Comment:
Since this link does not show in the REPL it might be a good idea to say something like
```suggestion
Run a command object, constructed with backticks (see the [Running External Programs](@ref) section in the manual).
```
and likewise for `success`.

Reply:
That makes sense, yes. I'll update the PR in a moment.